### DISCUSSION TITLE
关于图片保存成eps格式之后，出现打不开的情况

### DISCUSSION THREAD

**Discussion by chaoHnust** [weihuayi/fealpy#43] [Q&A]
魏老师，请问：直接使用fealpy 生成图片格式默认是png格式的，但是将格式改成eps之后，无法打开。如何解决这个问题

**Comment by weihuayi** [comment]
你是 windows 系统？有没有安装 eps 的阅读器？在邮件中应该把你生成的 eps 发给我，这是很有用的信息。
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Tue, Jul 6, 2021 04:08 AM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; 
Subject: &nbsp;[weihuayi/fealpy] 关于图片保存成eps格式之后，出现打不开的情况 (#38)

&nbsp;



 
魏老师，请问：直接使用fealpy 生成图片格式默认是png格式的，但是将格式改成eps之后，无法打开。如何解决这个问题

**Comment by scaomath** [comment]
@weihuayi 我觉得可以给fealpy设置一个[Discussion board](https://docs.github.com/en/discussions/quickstart)，这样这种非issue非bug的问题可以在Discussion里面问。

@chaoHnust 如果`plt.savefig`函数改成pdf文件没有问题，就是你本机无法读取eps。

**Comment by weihuayi** [comment]
@scaomath 是的，这是一个好主意，我设置一下。
